apps/homescreen/build/default-homescreens.json

```diff
@@ -0,0 +1,34 @@
+{
+  "homescreens": [
+    [
+      ["apps", "communications", "dialer"],
+      ["apps", "sms"],
+      ["apps", "communications", "contacts"],
+      ["apps", "email"],
+      ["apps", "search"]
+    ], [
+      ["apps/collection/collections", "social"],
+      ["apps/collection/collections", "games"],
+      ["apps/collection/collections", "music"]
+    ], [
+      ["apps", "camera"],
+      ["apps", "gallery"],
+      ["apps", "music"],
+      ["apps", "video"],
+      ["apps", "marketplace.firefox.com"]
+    ], [
+      ["apps", "calendar"],
+      ["apps", "clock"],
+      ["apps", "settings"],
+      ["apps", "fm"]
+    ], [
+      ["hosted_apps", "bugzillalite"],
+      ["hosted_apps", "facebook"],
+      ["hosted_apps", "twitter"],
+      ["apps", "costcontrol"]
+    ]
+  ],
+  "preferences": {
+    "grid.cols": 3
+  }
+}
```

Comment:
Why copy the file from verticalhomescreen and not instead create a file with just the data needed? Is it to make migration easier?